app/views/view_data/delivery_organisations/show.html.erb

```diff
@@ -33,14 +33,62 @@
     </div>
   </div>
 
+
   <%= render partial: "view_data/transactions_received_section" %>
 
+  <% [[:total_transactions,        :total],
+      [:online_transactions,       :online],
+      [:phone_transactions,        :phone],
+      [:paper_transactions,        :paper],
+      [:face_to_face_transactions, :face_to_face],
+      [:other_transactions,        :other]
+     ].each do |metric_name, short_metric_name| %>
+    <%= render partial: "view_data/charts/metric_chart", locals: {
+          metric: metric_name,
+          time_period: @metrics.time_period,
+          current_metrics: @current_by_metrics[metric_name],
+          previous_metrics: @previous_by_metrics[metric_name],
+          total_label: trxn_rx_metric_label(short_metric_name),
+          organisation: @delivery_organisation.name
+    } %>
+  <% end %>
+
   <hr/>
 
   <%= render partial: "view_data/transactions_processed_section" %>
 
+  <% [[:transactions_processed,     :total],
+      [:transactions_processed_with_intended_outcome, :with_intended_outcome]
+     ].each do |metric_name, short_metric_name| %>
+    <%= render partial: "view_data/charts/metric_chart", locals: {
+          metric: metric_name,
+          time_period: @metrics.time_period,
+          current_metrics: @current_by_metrics[metric_name],
+          previous_metrics: @previous_by_metrics[metric_name],
+          total_label: trxn_proc_metric_label(short_metric_name),
+          organisation: @delivery_organisation.name
+    } %>
+  <% end %>
+
   <hr/>
 
   <%= render partial: "view_data/calls_received_section" %>
```

Comment:
At the moment for delivery orgs and departments, it shows lots of empty charts because we can't check if something is applicable like we can for services. Do you think we should add something along the lines of
```
<% unless @calls_received.all? {|number_of_calls| number_of_calls == 0} %>
   <%= render partial: "view_data/calls_received_section" %>
    etc...
  <% else %>
  This service doesn't receive calls
<% end %>
```

and in the controller action

```
current_calls_received = @current_by_metrics[:calls_received]
previous_calls_received = @previous_by_metrics[:calls_received]

@calls_received = current_calls_received + previous_calls_received
```

I will try and think of a less lengthly way of doing it. Might be better for a separate PR but would be good to eliminate the non applicable charts for departments and delivery orgs.

Reply:
Actually, I added an ```is_applicable?(:online)``` to the service model, but this only works when viewing a single service.  For aggregated data, we've summed all the service data, so it might be that a dept has 2 services and it is not applicable for one, and is applicable for another.  So I'm not quite clear yet how we handle that.

Reply:
👍